ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt

```diff
@@ -152,8 +160,8 @@ class RedirectBindingVerifier(httpResponse: Response) : BindingVerifier(httpResp
      * 3.4.4.1 Deflate Encoding
      */
     @Suppress("ComplexMethod" /* Complexity due to nested `when` is acceptable */)
-    private fun decode(paramMap: Map<String, String>): Node {
-        val samlResponse = paramMap[SAML_RESPONSE]
+    fun decode(paramMap: Map<String, String>, isUrlEncoded: Boolean = true): Node {
```

Comment:
> Note: Response coming from SP2 endpoint is not url encoded.

Reply:
I think it's the way ktor handles it. I'll try to see if we can keep it URL encoded